pages/_app.js

```diff
@@ -10,20 +9,12 @@ import { TITLE, META_DESCRIPTION, META_IMAGE, URL } from "root/config";
 import ThemeProvider from "components/ThemeProvider";
 import * as gtag from "helpers/gtag";
 
-function MyApp({ Component: NextPage, pageProps }) {
+const App = ({ Component: NextPage, pageProps }) => {
```

Comment:
function to const comes from lint? Just wondering

Reply:
Nope I guess I just had it as const in the merge

Reply:
Ok, that's fine! Was just curious :)